drip/admin.py

```diff
@@ -36,6 +39,15 @@ class DripAdmin(admin.ModelAdmin):
     form = DripForm
     users_fields = []
 
+    def get_model_perms(self, request):
+        if not settings.ENABLE_QUERY_SET_RULE_PERMISSION:
+            return super(DripAdmin, self).get_model_perms(request)
+
+        if request.user.groups.filter(name='Drip Admin').exists():
+            return super(DripAdmin, self).get_model_perms(request)
```

Comment:
what would happen if it exists?

Reply:
Id the settings is set to enabled, and user doesn't have this Drip Admin group
The Drip will be hidden and at out end we handle the other admin.